h/sql_tasks/tasks/report/create_from_scratch/06_authority_activity/01_create_view.sql

```diff
@@ -91,15 +108,25 @@ CREATE MATERIALIZED VIEW report.authority_activity AS (
             -- it's available as primary data
             SUM(registering_users) OVER (
                 PARTITION BY timescale, authority_id ORDER BY period
-            ) AS total_users
+            ) AS total_users,
+            0 AS shared_annotations, 0 AS reply_annotations, 0 AS annotations
         FROM registering_users
 
         UNION ALL
 
         SELECT
             period, timescale, authority_id,
-            annotating_users, 0, 0
+            annotating_users, 0, 0,
+            0, 0, 0
         FROM annotating_users
+
+        UNION ALL
+
+        SELECT
+            period, timescale, authority_id,
+            0, 0, 0,
```

Comment:
Are these:
`annotating_users`
`registering_users`
`total_users`?

We could do with an alias in all of them, it's easy to follow the query once you have the structure in mind but reading bits in isolation is of course tricky.

Not something that needs to be done now.